__tests__/app.test.js

```diff
@@ -26,41 +22,47 @@ describe('/api/topics', () => {
             })
         })
     })
-    test('returns 404 if no topics found', ()=> {
-        return db.query(` DELETE FROM comments; DELETE FROM articles; DELETE FROM topics`).then(()=> {
-            return request(app)
-            .get('/api/topics')
-            .expect(404)
-            .then(({body}) => {
-                expect(body).toEqual({msg: "No Topics Found"})
-            })
-        })
-    })
-    test('returns 404 if table does not exist', ()=> {
-        return db.query(` DROP TABLE comments; DROP TABLE articles; DROP TABLE topics`).then(()=> {
-            return request(app)
-            .get('/api/topics')
-            .expect(404)
-            .then(({body}) => {
-                expect(body).toEqual({msg: "Table Does Not Exist"})
-            })
-        })
-    })
```

Comment:
This is a good deletion - we'd never really want to do queries directly from the test file